lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -511,7 +512,7 @@ module.exports = class MethodMappings {
           const element = this.getWebElement(webElement);
           await element.clear();
 
-          return null;
+          return {state: 'success', value: null};
```

Comment:
Any particular reason, we are adding `state` in the response

Reply:
Backward compatibility, Nightwatch used to return state property for `clearValue` command. You can observe this in tests